### PR TITLE
MF Tweak

### PR DESCRIPTION
**Pull request by brandon-binary**

### CODE REVIEW

File: src/javascript/_common/base/client_base.js [hunk 1]
```diff
@@ -312,6 +312,8 @@ const ClientBase = (() => {
         return (is_from_crypto ? !is_to_crypto : is_to_crypto);
     };
 
+    const isMFAccount = (loginid) => /^MF/.test(loginid || current_loginid);
```
Comment (negarn) [inline src/javascript/_common/base/client_base.js]: please don't test the login id. we don't hardcode these. we should always rely on landing company. you can instead use isAccountOfType('financial')

File: src/javascript/app/base/logged_in.js [hunk 2]
```diff
@@ -43,7 +43,10 @@ const LoggedInHandler = (() => {
             if (set_default) {
                 const lang_cookie = urlLang(redirect_url) || Cookies.get('language');
                 const language    = getLanguage();
-                redirect_url      = Client.defaultRedirectUrl();
+                // const current_loginid = getPropertyValue(response, ['authorize', 'loginid']);
```
Comment (negarn) [inline src/javascript/app/base/logged_in.js]: why is this here?

File: src/javascript/app/pages/trade/tradepage.js [hunk 3]
```diff
@@ -26,57 +26,57 @@ const TradePage = (() => {
...
-            Header.displayAccountStatus();
-            if (Client.get('is_virtual')) {
-                Header.upgradeMessageVisibility(); // To handle the upgrade buttons visibility
-                // if not loaded by pjax, balance update function calls TopUpVirtualPopup
-                if (State.get('is_loaded_by_pjax')) {
-                    BinarySocket.wait('balance').then(() => {
-                        TopUpVirtualPopup.init(State.getResponse('balance.balance'));
-                    });
-                }
+        if (!Client.isMFAccount()) {
```
Comment (negarn) [inline src/javascript/app/pages/trade/tradepage.js]: please use a return if client is financial instead so the indentation of the below lines don't change

**Comment by negarn** [comment]
@brandon-binary please resolve the conflict

**Comment by brandon-binary** [comment]
@negarn Fixed

File: src/javascript/app/base/binary_loader.js [hunk 4]
```diff
@@ -151,7 +154,7 @@ const BinaryLoader = (() => {
             return;
         }
 
-        const div_container = createElement('div', { class: 'logged_out_title_container', html: content.getElementsByTagName('h1')[0] || '' });
+        const div_container = createElement('div', { class: 'logged_out_title_container', html: Client.isAccountOfType('financial') ? '' : content.getElementsByTagName('h1')[0] });
```
Comment (negarn) [inline src/javascript/app/base/binary_loader.js]: the `|| ''` for the second condition should remain for pages that don't have header.
